cmd/bblfshd/main.go

```diff
@@ -94,7 +94,12 @@ func main() {
 	logrus.Infof("bblfshd version: %s (build: %s)", version, build)
 
 	r := buildRuntime()
-	d := daemon.NewDaemon(version, r, buildGRPCOptions()...)
+	grpcOpts, err := sdk.BuildGRPCOptions(*maxMessageSize)
+	if err != nil {
+		logrus.Errorln(err)
```

Comment:
Shouldn't this also exit the function? (you're using grpcOpts later even if `err != nil`)

Reply:
If that is allright, let's keep this discussion in https://github.com/bblfsh/bblfshd/pull/216#discussion_r235770197 and it would be good to have arguments there, why it should. I tried listing once that I see why it should not in there :)